src/main/java/seedu/address/model/person/Customer.java

```diff
@@ -0,0 +1,44 @@
+package seedu.address.model.person;
+
+import java.util.Set;
+
+import seedu.address.model.tag.Tag;
+
+
+
+/**
+ * Represents a Customer in the address book.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+
+public class Customer extends Person {
+
+    private static int idCount = 0;
```

Comment:
From memory I think static variable name needs to be all caps. Which is `IDCOUNT`